packages/ra-ui-materialui/src/input/SelectArrayInput.tsx

```diff
@@ -22,6 +22,7 @@ import {
 } from 'ra-core';
 import { InputHelperText } from './InputHelperText';
 import { FormControlProps } from '@mui/material/FormControl';
+import OutlinedInput from '@mui/material/OutlinedInput';
```

Comment:
Please put this import along the other ones from `'@mui/material'`

Reply:
Will do in the next iteration.